msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -450,40 +453,49 @@ export class MyBot extends TeamsActivityHandler {
 
 # [JSON](#tab/json)
 
-This is the message your bot receives when the bot is added to a team.
+This is the message your bot receives when the bot is added to a team channel.
```

Comment:
See comment above about bot being added to team vs channel.

Reply:
Updated as mentioned.